contrib/python-certifi/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "python-certifi"
+# only bump if there are any changes to the code
```

Comment:
technically it can always be bumped harmlessly, so not sure why this restriction